py-polars/polars/internals/lazy_functions.py

```diff
@@ -1484,6 +1514,40 @@ def apply(
     -------
     Expr
 
+    Examples
+    --------
+    >>> df = pl.DataFrame(
+    ...     {
+    ...         "a": [7, 2, 3, 4],
+    ...         "b": [2, 5, 6, 7],
+    ...     }
+    ... )
+    >>> df
+    shape: (4, 2)
+    ┌─────┬─────┐
+    │ a   ┆ b   │
+    │ --- ┆ --- │
+    │ i64 ┆ i64 │
+    ╞═════╪═════╡
+    │ 7   ┆ 2   │
+    │ 2   ┆ 5   │
+    │ 3   ┆ 6   │
+    │ 4   ┆ 7   │
+    └─────┴─────┘
+
+    calculate product of a
```

Comment:
I believe these small text snippets in between must have an empty line before and after to render properly. And it would be nice to have proper punctiation. You can use double backticks to make Sphinx render things as code.

```suggestion
    Calculate the product of column ``a``.
    
```

Reply:
How can I locally check if the formatting on the api reference is looking good? Also, still have some issues where I can't run `make fmt` without issues which doesn't help. So have to investigate what is missing so I can add that to the contributing document

Reply:
You can run `make html` from the `py-polars/docs` folder. That will build the docs locally.

Which command in the `make fmt` is causing trouble?